third_party/go-git-http/README.md

```diff
@@ -0,0 +1,79 @@
+go-git-http
+===========
+
+[![Build Status](https://travis-ci.org/AaronO/go-git-http.svg)](https://travis-ci.org/AaronO/go-git-http)
+
+A Smart Git Http server library in Go (golang)
```

Comment:
Why third_party rather than vendor?

Reply:
Modifying the code there unfortunately. The other library https://github.com/euforia/go-git-server was so much nicer, but it lacks a license and I did not manage to contact the author to add one (PR is open).